app/views/handlers/feedback.py

```diff
@@ -174,3 +200,41 @@ def is_limit_reached(session_data: SessionData) -> bool:
     @staticmethod
     def is_enabled(schema: QuestionnaireSchema) -> bool:
         return schema.get_submission().get("feedback")
+
+
+class FeedbackMetadata:
+    def __init__(self, feedback_count, form_type, language_code, region_code, tx_id):
+        self.feedback_count = feedback_count
+        self.form_type = form_type
+        self.language_code = language_code
+        self.region_code = region_code
+        self.tx_id = tx_id
+
+    def __call__(self) -> Mapping:
+        return {
+            "feedback_count": self.feedback_count,
+            "form_type": self.form_type,
+            "language_code": self.language_code,
+            "region_code": self.region_code,
+            "tx_id": self.tx_id,
+            "feedback_submission_date": datetime.utcnow().strftime("%Y-%m-%d"),
+        }
```

Comment:
This can still be simplied either of 2 ways.
1. set the datetime on initialisation (which could mean the time could be off by few millisecond but not really and issue):
```py
    def __init__(self, feedback_count, form_type, language_code, region_code, tx_id):
        self.feedback_count = feedback_count
        self.form_type = form_type
        self.language_code = language_code
        self.region_code = region_code
        self.tx_id = tx_id
        self.date_time = datetime.now(tz=tzutc()).isoformat()

    def __call__(self) -> Mapping:
        return vars(self)
```

2. Still use `vars` and update the dict.
```py
    def __call__(self) -> Mapping:
        metadata = vars(self)
        metadata["date_time"] = datetime.now(tz=tzutc()).isoformat()
        return metadata
```

Reply:
Doesn't match the key in the doc, isn't the format required, and is clearer as already it

Reply:
It reduces the references from 3 to 1.